packages/check-runtime/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Changelog
+
+## Unreleased
+
+Initial release
```

Comment:
I think it will be released straight away once this gets merged, if you've added it to the publish in CI